src/components/rating/_rating.scss

```diff
@@ -62,8 +61,8 @@ $includeHtml: false !default;
     }
 
     &--small {
-      @include fixText($rateSmallFontSize);
```

Comment:
Why is it removed? 
It should put the text in the middle of the available area in all browsers